### PR TITLE
DAT-39 package sync mediator

### PR DESCRIPTION
**Pull request by rcrichton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,24 @@ OpenHIM config package - for DATIM4U
...
+sync mediator which syncs site data between the openinfoman and dhis.
 
 The mediators are setup so that a request flows through them in this order:
 
 ```
 file-queue -> datim
 ```
 
+The site sync mediator is trigger separately via a polling channel and may also be
+trigger manually to esnure sites are insync. This mediator is setup to pull sites
```
Comment (nthfloor) [inline README.md]: typo - ensure, and triggered

**Comment by nthfloor** [comment]
Looks good @rcrichton Just the one typo change.

**Comment by rcrichton** [comment]
@nthfloor2 cool, fixed some typos.

**Comment by nthfloor** [comment]
@rcrichton awesome, looks great! :+1: 

I think you need to merge - doesn't look like I have permissions.

**Comment by rcrichton** [comment]
Oh right cool

**Pull request merged**